src/components/projects/lean-workflow/CommentList.tsx

```diff
@@ -350,24 +361,20 @@ export const CommentList: React.FC<Props> = ({ selected, editor }) => {
         commentFilter={commentFilter}
         setCommentFilter={setCommentFilter}
       />
-      <Pattern.Container>
+      <Container className={'comments-group'}>
         {items.map(([target, commentData]) => {
           // TODO: move this into a child component?
-          const isSelected =
-            (selected &&
-              (selected.node.attrs.id === target ||
-                selected.node.attrs.rid === target)) ||
-            false
           const selectedNoteData =
             commentFilter === Pattern.CommentFilter.ALL
               ? commentData
               : commentData.filter((note) => !note.comment.resolved)
           return (
-            <CommentTarget key={target} isSelected={isSelected}>
+            <CommentTarget key={target} isSelected={false}>
```

Comment:
I change the way we select comments by using the HTML class instead of props as this will help us to selected both comments in the inspector and the comment icon in the editor, and I don't want to remove props `isSelected` as we use it for other Component